ignite/pkg/chaincmd/runner/chain.go

```diff
@@ -236,6 +237,12 @@ func (r Runner) WaitTx(ctx context.Context, txHash string, retryDelay time.Durat
 
 // Export exports the state of the chain into the specified file
 func (r Runner) Export(ctx context.Context, exportedFile string) error {
+	// Make sure the path exists
```

Comment:
I added this because when I changed the previous caching strategy with files this folder was not always there.